railties/lib/rails/application/configuration.rb

```diff
@@ -291,6 +291,7 @@ def load_defaults(target_version)
             active_record.marshalling_format_version = 7.1
             active_record.run_after_transaction_callbacks_in_order_defined = true
             active_record.generate_secure_token_on = :initialize
+            active_record.attributes_for_inspect = :all
```

Comment:
Actually, I think we want this in the `7.2` (line 324) clause of this case statement, as this method will not exist in `7.1`.

Reply:
Got it! I also see this caused a linter issue by adding this. Working out how to fix that.